.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Not a huge deal but in the future be sure to add your `.vscode` directory with your launch config to your .gitignore